packages/core/admin/_internal/node/core/plugins.ts

```diff
@@ -49,18 +50,20 @@ export const getEnabledPlugins = async ({
   logger.debug('Core plugins', os.EOL, CORE_PLUGINS);
 
   for (const plugin of CORE_PLUGINS) {
-    const pkg = await getModule(plugin, cwd);
-
-    if (pkg && validatePackageIsPlugin(pkg)) {
-      /**
-       * We know there's a name because these are our packages.
-       */
-      const name = (pkg.strapi.name || pkg.name)!;
-
-      plugins[name] = {
-        name,
-        pathToPlugin: plugin,
-      };
+    if (plugin) {
```

Comment:
I needed to add this conditional because with the feature flag the content-releases plugin can be undefined @joshuaellis please let me know if you think it's okay 🤔

Reply:
I think that's fine yeah! Maybe just add a comment above saying something like "Some core plugins are activated via feature flags and as such might be undefined in the array"